llvm/test/CodeGen/AMDGPU/barrier-elimination-gfx12.ll

```diff
@@ -0,0 +1,55 @@
+; RUN: llc -global-isel=0 -mtriple=amdgcn -mcpu=gfx1200 < %s | FileCheck %s
```

Comment:
I strongly suggest to auto gen the check lines please, which can make other's life much easier in the future.

Reply:
Done.